src/pipe.js

```diff
@@ -163,6 +165,23 @@
         onloadcssdefined(loadCB);
         return ss;
     }
+    /*
+    * Custom Performance entries that can be added from fragments
+    * Its need because Browsers currently does not allow expose any API to add
+    * custom timing information to performance entries
+    */
+    function addPerfEntry(name, duration) {
+        entries.push({
+            name: name,
+            duration: Number(duration),
+            entryType: 'tailor',
+            startTime: perf.now() || Date.now() - perf.timing.navigationStart
```

Comment:
`navigationStart` is always available?

Reply:
good catch, i forgot to add a check for navigation timing.

Reply:
Fixed now, we should add the entry itself since it will result in skewed metrics.